.pylintrc

```diff
@@ -545,7 +545,7 @@ spelling-dict=en_US
 spelling-ignore-comment-directives=fmt: on,fmt: off,noqa:,noqa,nosec,isort:skip,mypy:
 
 # List of comma separated words that should not be checked.
-spelling-ignore-words=
+spelling-ignore-words=fixme, docstring
```

Comment:
We already have a mechanism to add words to ignore for spell check by adding them to this file (https://github.com/NEMRI-org/primo-optimizer/blob/main/.github/wordlist.txt). See Line 551 before

Please restore pylintrc file and instead add the words you wish to ignore in wordlist.txt

Reply:
`pylintrc` has been restored and `fixme` has been added to the `wordlist.txt`